client/src/test/java/se/fortnox/reactivewizard/client/HttpClientTest.java

```diff
@@ -961,6 +962,18 @@ public void shouldDeserializeVoidResult() {
         server.disposeNow();
     }
 
+    @Test
+    public void shouldDeserializeSimpleStringWithoutJsonQuotes() {
```

Comment:
Seems like you are missing a test case for when the response is `"null"` . Are there tests that cover strings that are quoted, so that is validated to be working?

Reply:
The "null" test is implemented in RestClientFactoryTest.shouldDeserializeNullToEmpty()